wagtail/admin/api/actions/create_alias.py

```diff
@@ -0,0 +1,52 @@
+from django.core.exceptions import ValidationError as DjangoValidationError
+from django.shortcuts import get_object_or_404
+from rest_framework import fields, status
+from rest_framework.exceptions import ValidationError
+from rest_framework.response import Response
+from rest_framework.serializers import Serializer
+
+from wagtail.api.v2.utils import BadRequestError
+from wagtail.core.actions.create_alias import CreatePageAliasAction, CreatePageAliasIntegrityError
+from wagtail.core.models import Page
+
+from .base import APIAction
+
+
+class CreatePageAliasAPIActionSerializer(Serializer):
+    parent_page_id = fields.IntegerField(required=False)
+    recursive = fields.BooleanField(default=False, required=False)
+    update_slug = fields.CharField(required=False)
+    update_locale = fields.CharField(required=False)
+    reset_translation_key = fields.BooleanField(default=True, required=False)
```

Comment:
I think the only time you would need to set this to ``False`` is when translating a page, but as mentioned above, that use case is covered by ``copy-for-translation`` so I think we can remove this